pjlib/src/pj/ssl_sock_ossl.c

```diff
@@ -1210,7 +1333,7 @@ static void ssl_destroy(pj_ssl_sock_t *ssock)
     /* Potentially shutdown OpenSSL library if this is the last
      * context exists.
      */
-    shutdown_openssl();
+    shutdown_openssl(ssock);
```

Comment:
as SSL socket instance count may reach zero in the middle of lib life time, so shutdown+reinit can be called multiple times and I think it will be inefficient (it is okay now because shutdown is actually no-op). An alternative is to schedule `shutdown_openssl()` using `pj_atexit()` so it will be called only when PJLIB is destroyed (note: must log when pj_atexit() return error!).

Reply:
okay, will use ```pj_atexit()```